hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStoreWithFSO.java

```diff
@@ -622,6 +621,40 @@ public void testRenameToAnExistingKey() throws Exception {
     }
   }
 
+  @Test
+  public void testCreateBucketWithBucketType() throws Exception {
+    String sampleVolumeName = UUID.randomUUID().toString();
+    String sampleBucketName = UUID.randomUUID().toString();
+    OzoneClient client = cluster.getClient();
+    ObjectStore store = client.getObjectStore();
+    store.createVolume(sampleVolumeName);
+    OzoneVolume volume = store.getVolume(sampleVolumeName);
+
+    // Case 1: Bucket Type: FSO
+    BucketArgs.Builder builder = BucketArgs.newBuilder();
+    builder.setBucketType(BucketType.FSO);
+    volume.createBucket(sampleBucketName, builder.build());
+    OzoneBucket bucket = volume.getBucket(sampleBucketName);
+    Assert.assertEquals(sampleBucketName, bucket.getName());
+    Assert.assertEquals(BucketType.FSO, bucket.getBucketType());
+
+    // Case 2: Bucket Type: OBJECT_STORE
+    sampleBucketName = UUID.randomUUID().toString();
+    builder.setBucketType(BucketType.OBJECT_STORE);
+    volume.createBucket(sampleBucketName, builder.build());
+    bucket = volume.getBucket(sampleBucketName);
+    Assert.assertEquals(sampleBucketName, bucket.getName());
+    Assert.assertEquals(BucketType.OBJECT_STORE, bucket.getBucketType());
+
+    // Case 3: Bucket Type: Empty
```

Comment:
Here, it says `Empty` but it is passing `DEFAULT` in the builder `builder.setBucketType(BucketType.DEFAULT);`.
Please remove `builder.setBucketType(BucketType.DEFAULT); `for verifying `Empty` case and you can add one more case for DEFAULT.

`// Case 4: Bucket Type: DEFAULT`

Reply:
Removed `builder.setBucketType(BucketType.DEFAULT);` and added `// Case 4: Bucket Type: DEFAULT`